flink-table/flink-table-planner/pom.xml

```diff
@@ -88,26 +88,26 @@ under the License.
 
 		<dependency>
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-table-api-java-bridge_${scala.binary.version}</artifactId>
+			<artifactId>flink-sql-parser</artifactId>
```

Comment:
Seems you don't change anything meaningful for this file?

Reply:
reverted